src/ui/utils.js

```diff
@@ -110,6 +113,60 @@ export function getBorderStyleLabels( t ) {
 	};
 }
 
+/**
+ * Returns a localized error string that can be displayed next to color (background, border)
+ * fields that have an invalid value.
+ *
+ * @param {module:utils/locale~Locale#t} t The "t" function provided by the editor
+ * that is used to localize strings.
+ * @returns {String}
+ */
+export function getLocalizedColorErrorText( t ) {
+	return t( 'The color is invalid. Try "#FF0000" or "rgb(255,0,0)" or "red".' );
+}
+
+/**
+ * Returns a localized error string that can be displayed next to length (padding, border width)
+ * fields that have an invalid value.
+ *
+ * @param {module:utils/locale~Locale#t} t The "t" function provided by the editor
+ * that is used to localize strings.
+ * @returns {String}
+ */
+export function getLocalizedLengthErrorText( t ) {
+	return t( 'The value is invalid. Try "10px" or "2em" or simply "2".' );
```

Comment:
@jodator I assumed here https://github.com/ckeditor/ckeditor5/issues/6098 will be merged soon.

Reply:
:crossed_fingers: hopefully - looking for a reviewers :P

Reply:
Oh - this one should be done on Monday. I though this one was about normalizing value passed to a command.